modules/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -537,7 +570,7 @@ protected function patternMatch($block, $orderedArgs, $keywordArgs) {
 			return true; // not having enough is handled above
 		} else {
 			$numMatched = $i + 1;
-			// greater than because default values always match
+			// greater than becuase default values always match
```

Comment:
Ah, looks like someone changed it downstream here in Jetpack.  Boo downstream changes.